src/core/generator/json/snapshots/tailcall__core__generator__json__query_generator__test__query_generator_with_query_params.snap

```diff
@@ -3,5 +3,5 @@ source: src/core/generator/json/query_generator.rs
 expression: config.to_sdl()
 ---
 type Query {
-  f1(is_verified: Boolean, q: Int): T1 @http(path: "/path", query: [{key: "q", value: "{{.args.q}}"}, {key: "is_verified", value: "{{.args.is_verified}}"}])
+  f1(is_verified: Boolean, q: Int): T1 @http(path: "/path", query: [{key: "q", value: "{{.args.q}}"}, {key: "is_verified", value: "{{.args.isVerified}}"}])
```

Comment:
The generated config won't work as .args.isVerified will always be null

There in no arg `isVerified` in the defined query